assets/css/twenty-seventeen.scss

```diff
@@ -102,7 +102,17 @@
 		color: #fff;
 	}
 }
-
+.colors-dark {
+	.woocommerce-pagination {
+		span.page-numbers.current {
+			background-color: #fff;
+		}
+		.next.page-numbers,.prev.page-numbers,a.page-numbers,span.page-numbers{
+			background: #505050;
+		}	
+		
+	}
+}
```

Comment:
I'm pretty sure all you have to do is make the text color darker for the non-hovered number buttons to fix the issue.